docs/data/xrootd/install-storage-element.md

```diff
@@ -380,6 +380,27 @@ root@host # service frm_xfrd start
 root@host # service frm_purged start
 ```
 
+
+### (Optional) Enabling xrootd-multiuser
+
+The `xrootd-multiuser` plugin makes the files written through XRootD to be owned by the writting user instaead of the `xrootd` user(default).
```

Comment:
```suggestion
The `xrootd-multiuser` plugin makes the files written through XRootD to be owned by the writing user instead of the `xrootd` user (default).
```